recipe/meta.yaml

```diff
@@ -35,9 +35,17 @@ requirements:
     - setuptools-scm
     - scikit-build
   run:
+    - ghapi
+    - nox
+    - packaging
+    - pandas
+    - requests
+    - scikit-build
+    - setuptools_scm
+    - spectrum
```

Comment:
@matthewfeickert you were right, it seems to hallucinate dependencies lol

Reply:
Yeah, given https://github.com/conda-forge/pyhf-feedstock/pull/22#issuecomment-1714584028 you might want to try `hint` instead of https://github.com/conda-forge/correctionlib-feedstock/blob/8e267de1b64f83ed9255875e63c95e2399ed681e/conda-forge.yml#L22